apps/explorer/src/app/routes/parties/id/components/party-accounts.tsx

```diff
@@ -23,7 +23,7 @@ export const PartyAccounts = ({ partyId }: PartyAccountsProps) => {
   const accounts =
     party?.accountsConnection?.edges?.filter((edge) => edge?.node) || [];
   return (
-    <div className="block min-h-44 h-60 4 w-full border-red-800 relative">
+    <div className="block min-h-44 4 w-full border-red-800 relative">
```

Comment:
Not sure if you actually want it but we have vega-red

Reply:
a) I didn't realise, thanks for pointing that out
b) I'm not even sure why this class is here, so I have removed the red border entirely.